packages/jwt/lib/src/models/jwt.dart

```diff
@@ -12,6 +13,38 @@ class Jwt {
     this.claims = const <String, dynamic>{},
   });
 
+  /// Decodes a JWT string of the format `header.payload.signature`. This does
+  /// _not_ perform any verification that the JWT is valid.
+  factory Jwt.unverifiedFromString(String string) {
```

Comment:
```suggestion
  factory Jwt.unverified(String string) {
```

nit but I don't think the `fromString` is needed 🤷

Reply:
Done